Sources/Core/Public/GTMSessionFetcher/GTMSessionUploadFetcher.h

```diff
@@ -133,6 +133,15 @@ typedef void (^GTMSessionUploadFetcherCancellationHandler)(GTMSessionFetcher *_N
 @property(atomic, copy) NSString *uploadMIMEType;
 @property(atomic, readonly, assign) int64_t chunkSize;
 @property(atomic, readonly, assign) int64_t currentOffset;
+@property(atomic, assign) double uploadRetryFactor;
+@property(atomic, assign)  NSTimeInterval nextUploadRetryInterval;
```

Comment:
Why does `nextUploadRetryInterval` need to be public? Could this just be an ivar? Or should we follow the fetcher behavior and store `lastUploadRetryInterval` and just calculate the next one via a method when it's needed?

Reply:
Lets just store the last one. For some reason I thought GTMSessionFetcher had stored the next one.